runatlantis.io/docs/server-configuration.md

```diff
@@ -496,7 +498,7 @@ Values are chosen in this order:
   ```bash
   atlantis server --tfe-token="xxx.atlasv1.yyy"
   # or (recommended)
-  ATLANTIS_TFE_TOKEN='xxx.atlasv1.yyy' atlantis server
```

Comment:
I think it makes sense to keep `atlantis server` here (and add it to the rest of the examples you've added) just to indicate the user needs to execute atlantis server with these env vars defined.

Reply:
I think the atlantis server obfuscates the fact that you should put these in a .env file and stop configuring tools on the commandline because it's harder to persist in a VCS somewhere :)

While I don't think you should put tokens into a .env file or a VCS, I further just think it obfuscates the fact that you'll be providing these on the commandline, and that 
`atlantis server --tfe-token="xxx.atlasv1.yyy"` and `ATLANTIS_TFE_TOKEN='xxx.atlasv1.yyy' atlantis server` are entirely identical.

You really shouldn't provide configuration on the commandline, other than secrets, which should just be passed as flags or, if you run it in docker, mounted in the environment when you run the container.

I would very much like to discuss this further if you disagree.

Reply:
yeah you're right, lets keep it as you have it!